docs/en/500k-hf-checklist.md

```diff
@@ -7,8 +7,8 @@ It is assumed you know what PoC1 and PoC2 is, how they differ, what optimized pl
 
 ### What To Do \*\*Now\*\* that  is released? (Everyone)
 
--   Make sure you have a current version of the wallet (2.2.6 or higher, currently released: )
--   Make sure you shut down all instances of pre- wallets, especially 1.2.9 or 1.3.6cg. These are **obsolete** and will no longer function after the fork.
+-   Make sure you have a current version of the wallet (2.4.2 or higher, currently released: )
```

Comment:
This was for the old fork, so that version is actually the correct one. Please adjust that so I can accept it.